src/components/PlayerDetails.js

```diff
@@ -1,21 +1,28 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
-const PlayerDetails = ({ player }) => {
-    if (!player) {
+const PlayerDetails = (props)=> {
+        const { selectPlayer } = props;
+
+    if (!selectPlayer) {
         return <h3>Select a Player</h3>
     }
 
     return (
         <div>
-            <h3>Player Stats</h3>            
-            <div>Field Goal Percentage:{player.FG_PCT}</div>
-            <div>Points: {player.PTS}</div>
-            <div>Rebounds: {player.REB}</div>
+            <h3>Stats</h3>
+            <h4>{selectPlayer.playerName}</h4>
+            <img src={`https://ak-static.cms.nba.com/wp-content/uploads/headshots/nba/1610612761/2019/260x190/${selectPlayer.playerId}.png`} />            
+            <div>Total Points: {selectPlayer.points}</div>
+            <div>Points per Game: {selectPlayer.ppg}</div>
+            <div>Blocks: {selectPlayer.blocks}</div>
+            <div>Steals: {selectPlayer.steals}</div>
```

Comment:
Can we use a destruct for all these values

`const { playerName, playerId, points, ppg, blocks, steals } = selectPlayer;`